clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTaskRepository.groovy

```diff
@@ -124,6 +126,8 @@ class JedisTaskRepository implements TaskRepository {
       pipe.expire(resultId, TASK_TTL)
       pipe.sync()
     }
+
+    log.debug("addResultObjects (resultId: ${resultId}, values: ${values})")
```

Comment:
It is the latter. Orca sees the task as complete, but the result objects it records are incomplete. On a subsequent read, the complete result objects are present.

An effort to deal with this on the Orca side is here: https://github.com/spinnaker/orca/pull/1051